python/perf-kernels/streamk/tune_streamk.py

```diff
@@ -542,6 +540,20 @@ def get_rocm_version():
 
 def main():
     args = parse_args()
+    # parse kernel file and kernel function name
+    module_name, kernel_name = args.kernel.split(',')
+    module_name = module_name.strip()
+    kernel_name = kernel_name.strip()
+    module = importlib.import_module(module_name)
+    kernel_func = getattr(module, kernel_name)
+
+    if not args.o:
+        if args.benchmark:
+            args.o = f"benchmarking_results_{kernel_name}.csv"
+        else:
+            default_name = get_default_tuning_result_filename()
+            args.o = f"{kernel_name}_{default_name}"
```

Comment:
this line makes an error to concatenate kernel name with the path with "_", should it be just default_name?

Reply:
Thanks @stephen-youn, fixed